src/models/global.rs

```diff
@@ -120,6 +138,8 @@ pub struct General {
     pub name: String,
     pub watched_version_branch: WatchedVersionBranch,
     pub show_opt_help: bool,
+    pub previous_version: String,
```

Comment:
Maybe use something like Option<versions::Version> from the [versions crate](https://docs.rs/versions/latest/versions/) ?

Reply:
Sounds nice, it looks very forgiving when parsing versions :D

Reply:
Haha true, it might be a bit too strict, I haven't had time to look at the version formatting hyperion.ng uses (although the versions crate seems to support a lot of different formats, including the appropriately-named [mess](https://docs.rs/versions/latest/versions/struct.Mess.html)).

Since we're not going to handle database/settings migrations anytime soon I feel like an Option<String> would be more than enough in this case.

Reply:
Option<String> that is?